tmpfiles.d/eos-companion-app-helper.conf

```diff
@@ -1,3 +1,5 @@
 # Type Path    Mode UID  GID  Age Argument
 d /var/lib/companion-app-helper 0755 companion-app-helper companion-app-helper -
 Z /var/lib/companion-app-helper 0755 companion-app-helper companion-app-helper -
+d /etc/avahi/services 0755 companion-app-helper companion-app-helper
+Z /etc/avahi/services 0755 companion-app-helper companion-app-helper
```

Comment:
Should this not be `root:avahi-service-writers`? (Or only change the group if possible.)